### DISCUSSION TITLE
How to create a SmartArt ornigazation chart into PowerPoint programmatically?

### DISCUSSION THREAD

**Discussion by ty05** [OfficeDev/Open-XML-SDK#1535] [Q&A]
Hi all,

I need to add function in the application that allow users to download their organization chart as SmartArt organization chart in PPT. 

I tried with Python library, but it does not create the org chart as SmartArt. 
https://python-pptx.readthedocs.io/en/latest/

It does not have to be Python, but could anyone please help and share sample codes? 
I found the doc below, but I am not familer with C# and not sure if it works or not.
https://learn.microsoft.com/en-us/dotnet/api/documentformat.openxml.drawing.diagrams.organizationchart?view=openxml-2.8.1

**Comment by ty05** [comment]
Never mind. I found a sample code in Java. 
https://dev.to/codesharing/java-how-to-create-a-smartart-graphic-in-powerpoint-53ec

**Comment by AlfredHellstern** [comment]
answered as per above
